sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py

```diff
@@ -0,0 +1,232 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint:disable=protected-access
+from typing import Optional
+from dateutil.tz import tzutc
```

Comment:
python-dateutil is not an allowed dependency, please use msrest.serialization.UTC
https://github.com/Azure/msrest-for-python/blob/76a414931470ada6210e926ac228767be3303195/msrest/serialization.py#L67